crates/ruff/resources/test/fixtures/perflint/PERF102.py

```diff
@@ -1,71 +1,101 @@
 some_dict = {"a": 12, "b": 32, "c": 44}
 
-for _, value in some_dict.items():  # PERF102
-    print(value)
 
+def f():
+    for _, value in some_dict.items():  # PERF102
+        print(value)
```

Comment:
Now that this rule relies on scoping, it's easier to put every case in its own scope. Otherwise, each case kind of pollutes every other case (if they're all in the module scope).